xml/System.Net/Cookie.xml

```diff
@@ -968,7 +968,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. The following characters are reserved and cannot be used for this property: semicolon, comma.  
+ The <xref:System.Net.Cookie.Value%2A> of a <xref:System.Net.Cookie> must not be `null`. If a `null` value is passed into the setter it will be replaced with an empty String. The following characters are reserved and cannot be used for this property: semicolon, comma. The setter won't throw an exception if invalid characters are passed into it but when you try to add this <xref:System.Net.Cookie> to a <xref:System.Net.CookieContainer> instance with the <xref:System.Net.CookieContainer.Add%2A> method the operation will fail and throw an exception.
```

Comment:
@rpetrusha You used a CookieCollection and not a CookieContainer as described in the Remarks. Code for reproduction:

```
var cookie = new Cookie("contoso", $"{DateTime.UtcNow}; Key=name,Value=SomeName;", "", "contoso.com");
			var coll = new CookieContainer();
			coll.Add(cookie);
```

I noticed that the current docs contain another discrepancy.

> So the following example constructor would succeed but when you try to add this Cookie to a CookieContainer instance with the Add or Add methods the operation will fail and throw an exception: 
System.Net.Cookie cookie = new System.Net.Cookie("contoso", "123,456", ""); 
However, the following constructor with these special characters escaped will create a Cookie that can be added to a CookieContainer instance: 
System.Net.Cookie cookie = new System.Net.Cookie("contoso", "\"123,456\""); 

Both of the calls will fail with an ArgumentException because the domain is an empty string:

```
if (cookie.Domain.Length == 0) {
                throw new ArgumentException(SR.GetString(SR.net_emptystringcall), "cookie.Domain");
            }
```

I will add "contoso.com" as the domain parameter to the sample.

Thanks for your feedback!